sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/QuorumReader.java

```diff
@@ -202,7 +203,7 @@ public Mono<StoreResponse> readStrongAsync(
                             case QuorumNotSelected:
                                 if (hasPerformedReadFromPrimary.v) {
                                     logger.warn("QuorumNotSelected: Primary read already attempted. Quorum could not be selected after retrying on secondaries.");
-                                    return Flux.error(new GoneException(RMResources.ReadQuorumNotMet));
+                                    return Flux.error(new GoneException(RMResources.ReadQuorumNotMet, HttpConstants.SubStatusCodes.READ_QUORUM_NOT_MET));
```

Comment:
I kind feel we should differentiate READ_QUORUM_NOT_MET vs READ_QUORUM_NOT_SELECTED. (but non blocking, just bring up the question, and also understand this is .Net behavior)

Reply:
Discussed offline, align with .Net behavior for now.